tempesta_fw/sock_clnt.c

```diff
@@ -404,7 +404,7 @@ tfw_listen_sock_start(TfwListenSock *ls)
 
 	inet_sk(sk)->freebind = 1;
 	sk->sk_reuse = 1;
-	r = ss_bind(sk, &addr->sa, tfw_addr_sa_len(addr));
+	r = ss_bind(sk, tfw_addr_sa(addr), tfw_addr_sa_len(addr));
```

Comment:
Changed `ss_connect()` and `ss_bind()` to accept `const TfwAddr *`.
`ss_bind()` now uses only `inet6_bind()`, but that should be fine, since `inet6_bind()` recognizes IPv4-mapped addresses and sets IPv4 source addresses just fine.